packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Next
+
+- Remove global scope CSS reset for `button` element in Button component.
+
 # 2.0.0
 
```

Comment:
Note: `@automattic/calypso-ui` 2.0.0 was never released (only an alpha).
After the rename to `@automattic/components`, we've published a `1.0.0-alpha.0`, meaning that the next version will be `@automattic/components` 1.0.0. This will be the first version to include `Card` and `Ribbon` (and `Button`, which wasn't added to the Changelog) :sweat_smile: 

Thus, I suggest the following Changelog modifications:


```suggestion
# @automattic/components 1.0.0

- Rename package from `@automattic/calypso-ui` to `@automattic/components`, reset version to 1.0.0.
- Add Button
```

(Since this is the first version to include `Button`, we actually don't need to mention the changes we're making to it.)

(I think the multi-line suggestion isn't quite working yet, I'll push a commit.)

Reply:
808c6654f1